procedures/katello/release.md.erb

```diff
@@ -27,6 +27,7 @@
 - [ ] Bump version:
   - [ ] Open a PR (or use cherry-pick PR) against the release branch which updates `lib/katello/version.rb` to <%= full_version %>
```

Comment:
Unrelated, but if you want a copy-paste instruction: `sed '/VERSION/ s/".\+"/"<%= full_version %>"/' lib/katello/version.rb`